opal/mca/common/ofi/common_ofi.c

```diff
@@ -554,11 +554,13 @@ static uint32_t get_package_rank(opal_process_info_t *process_info)
     }
 #endif
 
-    // Get the local peers
+    // Get the local peers - note there may not always be local peers,e.g. .singleton launch case
     OPAL_MODEX_RECV_VALUE(rc, PMIX_LOCAL_PEERS, &pname, &local_peers, PMIX_STRING);
     if (PMIX_SUCCESS != rc || NULL == local_peers) {
         // We can't find package_rank, fall back to procid
-        opal_show_help("help-common-ofi.txt", "package_rank failed", true);
+        if (10 <= opal_common_ofi_verbose_level) {
+            opal_show_help("help-common-ofi.txt", "package_rank failed", true, 10);
```

Comment:
I couldn't immediately tell what "10" does here by tracing the opal_show_help code, can you elaborate?

Reply:
Okay nvm, it's part of the help text which describes verbosity.